ui/app/AppLayouts/Chat/controls/community/CommunityDescriptionInput.qml

```diff
@@ -26,6 +26,10 @@ StatusInput {
             minLength: 1
             errorMessage: Utils.getErrorMessage(root.errors,
                                                 qsTr("community description"))
+        },
+        StatusRegularExpressionValidator {
+            regularExpression: /^$|^[a-zA-Z0-9\-_ ]+$/
```

Comment:
Can we extract this to Constants, instead of repeating it?

Reply:
I was trying to do it constant, but there is a problem... you can't combine constant with qml code
for example if we'll declare in Constant.qml:

```
property StatusRegularExpressionValidator test: StatusRegularExpressionValidator {
    regularExpression: /^$|^[a-zA-Z0-9\-_ ]+$/
    errorMessage: qsTr("Only letters, numbers, underscores, whitespaces and hyphens allowed")
}
```

and decide to use it in this file, the next record won't be build
```
validators: [
        StatusMinLengthValidator {
            minLength: 1
            errorMessage: Utils.getErrorMessage(root.errors,
                                                qsTr("community description"))
        },
        Constants.test
    ]
```
For sure, we can declare `StatusMinLengthValidator` as a property or a Component and set it, but I don't think that this will increase code readiness

Reply:
I think just moving the regular expression into utils and reusing it will be enough

Reply:
Yup, I meant just the regex itself, not the whole validator

Reply:
done

Reply:
cool, thanks